### ISSUE TITLE
property-sort-order does not pay attention to vendor prefixes

### ISSUE THREAD

**Issue by rugk** [sasstools/sass-lint#769]
>What version of Sass Lint are you using?

v1.8.2

> Please include any relevant parts of your configuration

``` yaml
  property-sort-order:
  - 1
  - order: smacss
```

> What did you do? Please include the actual source code causing the issue.

```scss
@mixin transform($transform) {
  -webkit-transform: $transform;
  -moz-transform: $transform;
  -ms-transform: $transform;
  -o-transform: $transform;
  transform: $transform;
}
```

> What did you expect to happen?

No error from `property-sort-order`as there are only some vendor prefixes...

Even with `ignore-custom-properties` set to true it does not work.

> What actually happened? Please include any error messages given to you by Sass Lint.

> property-sort-order: Expected `transform`, found `-webkit-transform`at line 28 col 3

> If you're using a IDE plugin have you tried the CLI too?

No, Atom plugin.
